popget/client.py

```diff
@@ -37,7 +37,7 @@ def method_factory(endpoint, client_method_name):
         deserialized JSON data.
     """
     def _prepare_request(base_url, _request_kwargs=None, **call_kwargs):
-        # type: (str, Optional[Dict], **Any) -> Tuple[str, Dict[str, Dict]]
+        # type: (str, dict | None, **Any) -> tuple[str, dict[str, dict]]
```

Comment:
the comments could be converted to actual type annotations if migrating to Py3 now 👍